dbt/include/redshift/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -0,0 +1,44 @@
+{% macro redshift__get_columns_spec_ddl() %}
+  {#- loop through user_provided_columns to create DDL with data types and constraints -#}
+  {%- if config.get('constraints_enabled', False) -%}
+    {%- set user_provided_columns = model['columns'] -%}
+    {%- set primary_keys = [] -%}
+    {%- set ddl_lines = [] -%}
+
+    {%- for i in user_provided_columns -%}
+      {%- set col = user_provided_columns[i] -%}
+      {%- set constraints = col['constraints'] -%}
+      {%- set ns = namespace(not_null_line = '') -%}
+
+      {%- for constraint in constraints -%}
+        {%- if constraint == 'primary key' -%}
+          {%- do primary_keys.append(col['name']) -%}
+        {%- elif constraint == 'not null' %}
+          {%- set ns.not_null_line = " not null" -%}
+        {%- endif -%}
+      {%- endfor -%}
+
+      {%- set not_null_line = " not null" if not_null_col else "" -%}
+
+      {%- set check = col['check'] -%}
```

Comment:
```suggestion
      {%- set check = col['constraints_check'] -%}
```
Do a find and replace on all check configs in all files to be constraints_check by name.